infra/hoogle_server.tf

```diff
@@ -18,16 +18,32 @@ resource "google_compute_firewall" "hoogle" {
   }
 }
 
+locals {
+  h_clusters = [
+    {
+      suffix         = "-blue",
+      ubuntu_version = "1604",
```

Comment:
should we check this in with `2004` given that we already deployed #9344 so next time we use this infrastructure, we definitely don’t want either blue or green to be 1604? Or are you keeping it so we can rollback?

Reply:
In the general case, I don't know what will change next time (in this case I have a strong suspicion it's going to be related to #9352). I'm not entirely decided whether we should keep this specific field at all vs. defining the fields based on the change being made. This is still a bit new to me.